HousingSearchApi/V2/Gateways/SearchGateway.cs

```diff
@@ -17,22 +18,49 @@ public SearchGateway(IElasticClient elasticClient)
     public async Task<SearchResponseDto> Search(string indexName, SearchParametersDto searchParams)
     {
         var searchResponse = await _elasticClient.SearchAsync<object>(s => s
-                .Index(indexName)
-                .Query(q => q
-                    .SimpleQueryString(qs => qs
-                        .Fields("*")
-                        .Query(searchParams.SearchText)
+            .Index(indexName)
+            .Query(q => q
+                .Bool(b => b
+                    .Should(
+                        MultiMatchSingleField(searchParams.SearchText, boost: 6),
+                        MultiMatchAcrossFields(searchParams.SearchText, boost: 2)
```

Comment:
Quite curious for what are these boost values.
Could you link to the documentation of this package?

Reply:
Boost is a relative weight between should conditions.